mobile-app/components/Board.tsx

```diff
@@ -0,0 +1,91 @@
+import * as React from 'react';
+import Svg from 'react-native-svg';
+import PlayableCell from './PlayableCell';
+import BorderCell from './BorderCell';
+import { GameState } from '../../web-app/src/common/gameState';
+import { CellType } from "../utils";
+
+const CELL_STROKE_COLOR = "white";
+const CELL_SIZE = 20;
+const DISTANCE_BETWEEN_HEXAGONE_PARALLEL_SIDES = Math.sqrt(3);
+
+interface BoardData {
+  gameState: GameState;
+}
+
+export default function Board(props: BoardData) {

```

Comment:
```suggestion
export default function Board({ gameState } : BoardData) {

```

same than below, it's easier to read with this